upgrade-monitoring-services.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Upgrade Cluster Monitoring Services
+summary: Learn how to upgrade the Prometheus, Grafana, and Alertmanager monitoring services for your TiDB cluster.
+---
+
+# Upgrade TiDB Cluster Monitoring Services
+
+When you deploy a TiDB cluster using TiUP, TiUP automatically deploys monitoring services (such as Prometheus, Grafana, and Alertmanager) for the cluster. If you scale out this cluster, TiUP also automatically adds monitoring configurations for newly added nodes during the scaling.
+
+Pay attention that TiUP uses its own configurations to override the configurations of the monitoring services. If you directly upgrade the monitoring services by replacing their configuration files, any subsequent operations such as `deploy`, `scale-out`, `scale-in`, or `reload` on the cluster might overwrite your upgrade, leading to errors. To upgrade Prometheus, Grafana, and Alertmanager, you can follow the upgrade steps in this document.
+
+> **Note:**
+>
+> - If your monitoring services are not deployed and managed by TiUP, you can directly upgrade them without referring to this document.
+> - The TiDB compatibility with newer versions of monitoring services has not been not tested, so it is possible that some features might not work as expected after the upgrade. For any issues, you can create an [issue](https://github.com/pingcap/tidb/issues) on GitHub.
```

Comment:
So if people upgrade they don't have a supported version anymore?

Reply:
The official supported version is the one in the tidb package. Monitoring is provided by 3rd party, we do not have resources to test the different versions . We will do some fix if users found any high impact issues.

Reply:
The problem I see with this is with vulnerabilities that are fixed upstream but not in the version that is in the tidb package.  Then the user could upgrade, but then they'll lose support.